index.html

```diff
@@ -2,12 +2,37 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <!--Bootstrap Css -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
+      crossorigin="anonymous"
+    />
+
+    <!-- font awsome cdn -->
```

Comment:
While using Bootstrap and Font Awesome via a CDN is fine, it's not ideal for a Vite project where module bundling is a key feature. Instead, you should import these libraries directly into your project using npm, which makes version management easier and enables tree-shaking to avoid loading unnecessary CSS/JS.

Using npm helps lock versions and prevents unintentional updates.
Vite can bundle only the necessary parts of these libraries, reducing load times.